Fabric.Authorization.AccessControl/src/app/services/interceptors/index.ts

```diff
@@ -2,17 +2,25 @@
 import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { FabricHttpRequestInterceptorService } from './fabric-http-request-interceptor.service';
 import { FabricHttpErrorHandlerInterceptorService } from './fabric-http-error-handler-interceptor.service';
+import { FabricHttpFakeDiscoveryInterceptorService } from './fabric-http-fake-discovery-interceptor.service';
+import { environment } from '../../../environments/environment'
 
 /** Http interceptor providers in outside-in order */
-export const httpInterceptorProviders = [
-  {
+export const httpInterceptorProviders = [];
+if (!environment.production) {
+  httpInterceptorProviders.push({
     provide: HTTP_INTERCEPTORS,
-    useClass: FabricHttpRequestInterceptorService,
+    useClass: FabricHttpFakeDiscoveryInterceptorService,
     multi: true
-  },
-  {
-    provide: HTTP_INTERCEPTORS,
-    useClass: FabricHttpErrorHandlerInterceptorService,
-    multi: true
-  }
-];
+  });
+}
+httpInterceptorProviders.push({
```

Comment:
Should this block be wrapped in an `else`?

Reply:
No - we always want these interceptors added, we just want to conditionally add the fake interceptor to allow mocking only when we don't build in production mode.